LiteCore/Storage/SQLiteKeyStore.cc

```diff
@@ -390,17 +409,18 @@ namespace litecore {
                 }
                 sql << ")";
                 db().exec(sql.str(), LogLevel::Info);
-
+                db().exec("INSERT OR REPLACE INTO kv_fts_map (alias, expression) VALUES (\"" + alias +
```

Comment:
Take out `OR REPLACE` — by this point, there shouldn't already be an entry with this alias (since we were supposed to have detected that up above.)